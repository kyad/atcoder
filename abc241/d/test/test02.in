12
1 2000000000000
1 1000000000000
1 3000000000000
1 2000000000000
3 2000000000000 1
3 2000000000000 2
3 2000000000000 3
3 2000000000000 4
2 10000000000000 5
1 100000000000
2 10000000000000 5
2 2000000000000 1
