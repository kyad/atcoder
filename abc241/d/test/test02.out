2000000000000
2000000000000
3000000000000
-1
-1
100000000000
2000000000000
